src/daemon/rpc_command_executor.cpp

```diff
@@ -378,9 +378,11 @@ bool t_rpc_command_executor::show_status() {
     % (unsigned)hfres.version
     % get_fork_extra_info(hfres.earliest_height, ires.height, ires.target)
     % (hfres.state == cryptonote::HardFork::Ready ? "up to date" : hfres.state == cryptonote::HardFork::UpdateNeeded ? "update needed" : "out of date, likely forked")
-    % (unsigned)ires.outgoing_connections_count % (unsigned)ires.incoming_connections_count
-    % (unsigned int)floor(uptime / 3600.0)
-    % (unsigned int)floor(fmod(uptime, 3600.0) / 60.0)
+    % (unsigned)ires.outgoing_connections_count
+    % (unsigned)ires.incoming_connections_count
+    % (unsigned int)floor(uptime / 60.0 / 60.0 / 24.0)
+    % (unsigned int)floor(fmod((uptime / 60.0 / 60.0), 24.0)
+    % (unsigned int)floor(fmod((uptime / 60.0), 60.0)
```

Comment:
Unbalanced braces on the last 2 lines.  4 left, 3 right.  Cause error caught by compiler.
/home/tdprime/bitmonero/src/daemon/rpc_command_executor.cpp: In member function ‘bool daemonize::t_rpc_command_executor::show_status()’:
/home/tdprime/bitmonero/src/daemon/rpc_command_executor.cpp:386:38: error: invalid operands of types ‘double’ and ‘unsigned int’ to binary ‘operator%’
     % (unsigned int)fmod(uptime, 60.0)
     ^